bin/jsondiff.py

```diff
@@ -0,0 +1,113 @@
+import os
+import sys
+import argparse
+import diff
+import deepdiff
+
+def add_arguments():
+    """Returns the arguments provided by the user and parsed by the Argument Parser"""
+    parser = argparse.ArgumentParser(description='Provide two json files (.json) to check for differences', add_help=False)
+    options = parser.add_argument_group(title='Options')
+    options.add_argument('json_files',
+                         metavar='<json_file, json_file>',
+                         help='Specify two json files',
+                         type=validate_json_extension,
+                         nargs=2)
+    # Note: nargs='?' and const=value is for 0 or 1 arguments supplied. If 0 arguments provided, the const value will be used
+    options.add_argument('--rel_tol',
+                         metavar='<relative_error>',
+                         dest='relative_error',
+                         help='Specify the maximum relative error (default 1e-8)',
+                         type=float,
+                         nargs='?', 
+                         const=1e-8)
```

Comment:
I think you  can use `default = None` and get rid of the check below with sys.argv.